### PR TITLE
Login Epilogue: move connect site option

### PR DESCRIPTION
**Pull request by ScoutHarris**
Ref #13413 

This moves the 'Connect another site' option.
- 'Connect another site' button removed.
- 'Connect another site' row added to the bottom of the sites list.
- The 'Continue' button has been renamed to 'Done'.

Not yet updated:
- Button area background.
- iPad specific layout.
- Accessibility.

---
To test:
- Log into the app.
- Verify 'Connect another site' appears at the bottom of the list, and launches the site connection flow.
- Verify 'Connect another site' looks like the Zeplin design, or that on #13413 .

|  short_list_light |  short_list_dark |
|--------|-------|
|  long_list_light |  long_list_dark |
|  no_sites_light |  no_sites_dark |
---
PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by ScoutHarris** [comment]
@mindgraffiti I finally noticed I broke the UI tests. I'm looking at it.

File: WordPress/Classes/ViewRelated/NUX/LoginEpilogueTableViewController.swift [hunk 1]
```diff
@@ -66,6 +84,12 @@ extension LoginEpilogueTableViewController {
             }
         }
 
+        // Add one for Connect Site if there are no sites from blogDataSource.
+        if adjustedNumberOfSections == 0 && showConnectSite {
+            adjustedNumberOfSections += 1
```
[the remark this answers is not in the document]
Comment (ScoutHarris) [reply WordPress/Classes/ViewRelated/NUX/LoginEpilogueTableViewController.swift]: I don't see that having a constant just for adding 1 is beneficial. The 1 doesn't represent anything in particular, other than simply increasing the section count.

File: WordPress/Classes/ViewRelated/NUX/LoginEpilogueTableViewController.swift [hunk 2]
```diff
@@ -75,30 +99,42 @@ extension LoginEpilogueTableViewController {
...
+                return UITableViewCell()
+            }
+            if let info = epilogueUserInfo {
+                cell.stopSpinner()
+                cell.configure(userInfo: info)
+            } else {
+                cell.startSpinner()
+            }
+
+            return cell
```
[the remark this answers is not in the document]
Comment (ScoutHarris) [reply WordPress/Classes/ViewRelated/NUX/LoginEpilogueTableViewController.swift]: I'm not sure I agree this is better. Currently, both the `if` and `else` fall through to the same `return`, which is more clear what's happening. In your example, there is an additional (and I think unnecessary) `return`. I don't feel a `guard` is necessary in this case.

[the remark this answers is not in the document]
Comment (mindgraffiti) [reply WordPress/Classes/ViewRelated/NUX/LoginEpilogueTableViewController.swift]: That’s fine, it’s all optional 🙂

File: WordPress/Classes/ViewRelated/NUX/LoginEpilogueTableViewController.swift (hunk 1 above)
[the remark this answers is not in the document]
Comment (mindgraffiti) [reply WordPress/Classes/ViewRelated/NUX/LoginEpilogueTableViewController.swift]: That’s fair. I view nits as optional, not a hard requirement 🙂

**Comment by ScoutHarris** [comment]
Hey @mattmiklic .

> If the user has no sites, could we drop the "My Sites" header?

Can do!

**Pull request merged**